packages/jest-cli/src/Growler.js

```diff
@@ -0,0 +1,98 @@
+/**
+* Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+*
+* @flow
+*
+* This source code is licensed under the BSD-style license found in the
+* LICENSE file in the root directory of this source tree. An additional grant
+* of patent rights can be found in the PATENTS file in the same directory.
+*/
+'use strict';
+
+import type {AggregatedResult} from 'types/TestResult';
+
+const growl = require('growl');
+const path = require('path');
+const util = require('util');
+
+const isDarwin = process.platform === 'darwin';
+
+const APP_NAME = 'Jest';
+
+const ICON = '/assets/jest_logo.png';
+
+const NOTIFICATIONS = {
+  success: {
+    text: (isDarwin ? '✅ ' : '') + '%d tests passed',
+    options: {
+      title: '%d%% Passed',
+      image: ICON,
+      name: APP_NAME,
+      sticky: false,
+      priority: 0,
+    },
+  },
+  failure: {
+    text: (isDarwin ? '⛔️ ' : '') + '%d of %d tests failed',
+    options: {
+      title: '%d%% Failed',
+      image: ICON,
+      name: APP_NAME,
+      sticky: false,
+      priority: 1,
+    },
+  },
+};
+
+type Notification = {
+  text: string,
+  options: {
+    title: string,
+    image: string,
+    name: string,
+    sticky: boolean,
+    priority: ?number;
+  }
+};
+
+class Growler {
```

Comment:
Can we add a test for this class? I wanna make sure we won't be breaking this feature :)

We can just `jest.mock('grown')` and check if it has been called with the proper args.